cpp/log/cert.h

```diff
@@ -30,6 +30,7 @@ class Cert {
   // Will return null if |x509| is null.
   static std::unique_ptr<Cert> FromX509(ScopedX509 x509);
 
+  Cert() = delete;
```

Comment:
While redundant (the default constructor is suppressed by the constructor on line 201), I suppose this makes it easier to see at a glance that there is no default constructor.